src/simulators/qasm/qasm_controller.hpp

```diff
@@ -82,6 +82,17 @@ namespace Simulator {
  *      parallel circuit or shot execution is enabled this will only
  *      use unallocated CPU cores up to max_parallel_threads. [Default: 100]
  *
+* From MatrixProductState::State class
+ * - "mps_approximation_threshold" (double): Set the cutoff value for the
```

Comment:
I think this should be called `"matrix_product_state_bond_truncation_threshold"`, and should always be used (with a default value) regardless of whether the bond dimension is also specified. If you don't want any approximation this can be set to 0 by default.